src/components/skills-quiz/CourseCard.jsx

```diff
@@ -25,124 +26,79 @@ const CourseCard = ({
     };
   }, [course]);
 
-  const loadingCard = () => (
-    <Card>
+  const primaryPartnerLogo = partnerDetails.primaryPartner && partnerDetails.showPartnerLogo ? {
+    src: partnerDetails.primaryPartner.logoImageUrl,
+    alt: partnerDetails.primaryPartner.name,
+  } : undefined;
+
+  const handleCardClick = () => {
+    if (isLoading) {
+      return;
+    }
+    history.push(linkToCourse(course, slug, uuid));
+  };
+
+  return (
+    <Card
+      isClickable
+      isLoading={isLoading}
+      onClick={handleCardClick}
+      data-testid="skills-quiz-course-card"
+    >
       <Card.ImageCap
-        as={Skeleton}
-        duration={0}
+        src={course.cardImageUrl || course.originalImageUrl}
+        srcAlt=""
```

Comment:
The usage of `alt` text on an image should be intentional and convey the semantic meaning of the image, if it's core to the experience. If the image is purely decorative (i.e., it doesn't really semantically mean anything to the user), an empty `alt` text is permissible.

In this case, the image used on the card associated with the content is just decorative and it's not really all that important for the user to know what's in the image.

From some [official docs](https://www.w3.org/WAI/tutorials/images/decorative/):

> In these cases, a null (empty) alt text should be provided (alt="") so that they can be ignored by assistive technologies, such as screen readers. Text values for these types of images would add audible clutter to screen reader output or could distract users if the topic is different from that in adjacent text. Leaving out the alt attribute is also not an option because when it is not provided, some screen readers will announce the file name of the image instead.